libs/repositories/src/SimulationRepository/SimulationRepository.ts

```diff
@@ -16,7 +16,7 @@ export interface SimulationData {
   // { [address: string]: { [token: string]: balanceDiff: string } }
   // example: { '0x123': { '0x456': '100', '0xabc': '-100' } }
   cumulativeBalancesDiff: Record<string, Record<string, string>>;
-  gasUsed: string;
+  gasUsed?: string;
```

Comment:
Is this needed? Seems unrelated since the issue is when `transaction` is null

Reply:
This is needed because of the change on the line 65 of `libs/repositories/src/SimulationRepository/SimulationRepositoryTenderly.ts` file